mc/gui/intro_dlg.py

```diff
@@ -17,35 +18,40 @@ class IntroDlg(QtWidgets.QDialog):
     def __init__(self):
         super().__init__()
 
-        self.setGeometry(300, 300, 550, 450)
-
-        vbox_l2 = QtWidgets.QVBoxLayout()
-        self.setLayout(vbox_l2)
         self.wizard_qsw_w3 = QtWidgets.QStackedWidget()
-        vbox_l2.addWidget(self.wizard_qsw_w3)
+        self.prev_qpb = QtWidgets.QPushButton(PREV_STR)
+        self.next_qpb = QtWidgets.QPushButton(NEXT_STR)
+
+        self.initUI()
+
+    def initUI(self):
```

Comment:
Can you change this to `init_ui`? There's a naming convention i like for using lower case letters for function names (different from Java where i come from but i've slowly learned to adapt)